New package: SymbolicRegressionGPU v1.5.2

JuliaRegistrator opened the pull request:
- Registering package: SymbolicRegressionGPU
- Repository: https://github.com/x66ccff/SymbolicRegressionGPU.jl
- Created by: @x66ccff
- Version: v1.5.2
- Commit: 2744420e3680a760953f158e558ef7a5e3567fcb
- Reviewed by: @x66ccff
- Reference: https://github.com/x66ccff/SymbolicRegressionGPU.jl/issues/19#issuecomment-2611298878
- Description: PSRN (Parallel Symbolic Regression Network) enhanced SymbolicRegression.jl for fast, large-scale parallel symbolic evaluations on GPUs
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598251ad994c53f5ce2b72148b79a39074ac0c71228f013864a1d4a1ee2b951b67849565a4fa36c071cddfb417d99e3ac20521790759c6f048da0189855c6442d6be932bb14ae5ec03e24952c7b008d4eafc21c09118a8ca955043583a8e9307a58e2dea9bcde04112fcedb142f7ba203070b56e5eada14622b366f16e32e9042cd -->

x66ccff commented:
stop

x66ccff commented:
[noblock] I use `THArrays.jl` as a submodule in this package, however this requires CUDA and cmake to build this package.

x66ccff commented:
[noblock] While SymbolicRegressionGPU.jl originated as a fork of SymbolicRegression.jl, it integrates the [PSRN](https://github.com/intell-sci-comput/PTS) framework to accelerate symbolic regression, distinguishing it from a mere copy.

JuliaRegistrator committed:
New package: SymbolicRegressionGPU v1.5.2

UUID: 15561277-8ce4-40f2-9422-5d7a3ff74b8d
Repo: https://github.com/x66ccff/SymbolicRegressionGPU.jl.git
Tree: 121a929aa521530b48e29f866b02d7670d16635c

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
